backend/src/test/java/com/pickpick/message/application/ReminderServiceTest.java

```diff
@@ -61,293 +67,361 @@ class ReminderServiceTest {
+                List<Long> expectedIds = extractFutureFromTargetOrderedByRemindDateIds(bomReminders, target, 20);
+
+                assertThat(foundReminders).extracting("id").containsExactlyElementsOf(expectedIds);
+            }
+        }
+
+        @DisplayName("요청 파라미터에 리마인더 id와 count가 있다면")
+        @Nested
+        class reminderIdAndCountInParams {
+
+            Reminder target = bomReminders.get(5);
+            int count = 3;
+
+            ReminderFindRequest request = ReminderFindRequestFactory.reminderIdAndCount(target.getId(), count);
+
+            @DisplayName("해당 리마인더보다 미래 시간의 리마인더를 count개 조회한다")
+            @Test
+            void findCountRemindersSetRemindedAfterTarget() {
+                List<ReminderResponse> foundReminders = reminderService.find(request, bom.getId()).getReminders();
+                List<Long> expectedIds = extractFutureFromTargetOrderedByRemindDateIds(bomReminders, target, count);
+
+                assertThat(foundReminders).extracting("id").containsExactlyElementsOf(expectedIds);
+            }
+        }
+
+        @DisplayName("messageId와 memberId로 단건 조회한다")
+        @Test
+        void findOne() {
+            Reminder target = bomReminders.get(5);
+            ReminderResponse response = reminderService.findOne(target.getMessage().getId(),
+                    target.getMember().getId());
+
+            assertThat(target.getId()).isEqualTo(response.getId());
+        }
+
+        @DisplayName("messageId와 memberId가 일치하는 리마인더가 없다면 예외가 발생한다")
+        @Test
+        void findOneThrowsException() {
+            Member kkojae = members.save(KKOJAE.create());
+            Message target = noticeMessages.get(0);
+
+            assertThatThrownBy(() -> reminderService.findOne(target.getId(), kkojae.getId()))
+                    .isInstanceOf(ReminderNotFoundException.class);
+        }
+
+        private List<Message> createAndSaveMessages(final Channel channel, final Member member) {
+            List<Message> messagesInChannel = Arrays.stream(MessageFixtures.values())
+                    .map(messageFixture -> messageFixture.create(channel, member))
+                    .collect(Collectors.toList());
+
+            for (Message fixture : messagesInChannel) {
+                messages.save(fixture);
+            }
+
+            return messagesInChannel;
+        }
+
+        private List<Reminder> saveRemindersAtDifferentTimes(final Member member, final List<Message> messages) {
+            List<Reminder> savedReminders = new ArrayList<>();
+            int totalSize = messages.size();
+
+            for (int i = 0; i < totalSize; i++) {
+                LocalDateTime remindDateTime = LocalDateTime.now().plusHours(i + 1);
+                Reminder reminder = reminders.save(
+                        new Reminder(member, messages.get(totalSize - i - 1), remindDateTime));
+                savedReminders.add(reminder);
+            }
+
+            return savedReminders;
+        }
```

Comment:
```suggestion
            LocalDateTime remindDateTime = LocalDateTime.now();
            return messages.stream()
                    .map(message ->
                            reminders.save(new Reminder(member, message, remindDateTime.plusDays(message.getId()))))
                    .collect(Collectors.toList());
        }
```

여기도 반복문을 스트림으로 바꿔서 살짝 짧게 만들어봤어요!. 
remindDateTime.plusDays() 파라미터로 숫자를 넣고 싶었는데, 방법을 못찾아서 message.getId() 로 대체해봤습니다. tearDown() 때 databaseClean 해주고 이때 primarykey 시작도 다시 초기화해주니 괜찮을까 싶다가도 getId()를 사용한다는거 자체가 살짝 걸리네요!

단순 의견이니 반영하지 않으셔도 됩니다1!!

Reply:
여기도 위와 동일한 이유로 반복문 사용이 낫다고 생각합니다..!